CHANGELOG.md

```diff
@@ -31,14 +31,19 @@
   [**@Sherwin-14**](https://github.com/betolink),
   [**@mfisher87**](https://github.com/mfisher87)
 
-
-
 ### Fixed
 
 - Removed Broken Link "Introduction to NASA earthaccess"
   ([#779](https://github.com/nsidc/earthaccess/issues/779))
   ([**@Sherwin-14**](https://github.com/Sherwin-14))
 
+### Removed
+
+- `binder/` directory as we no longer need a special [binder](https://mybinder.org)
+  environment with the top-level `environment.yml` introduce in
+  [#733](https://github.com/nsidc/earthaccess/issues/733)
+  ([@jhkennedy](https://github.com/jhkennedy))
```

Comment:
_If_ we keep this entry, then I suggest the following minor tweaks:

```suggestion
- Remove `binder/` directory, as we no longer need a special [binder](https://mybinder.org)
  environment with the top-level `environment.yml` introduced in
  [#733](https://github.com/nsidc/earthaccess/issues/733)
  ([@jhkennedy](https://github.com/jhkennedy))
```

based upon the following reasonings:

- Even though this is listed within the **Removed** section, per section [2.4.1](https://common-changelog.org/#241-change), each entry should be self-describing; hence the addition of "Remove" at the beginning of the bulleted item.
- Also per section 2.4.1, use imperative mood with present-tense verbs (hence "Remove", not "Removed")
- Further (now I'm getting really nit-picky), the **Removed** section should appear above the **Fixed** section, per the ordering specified in [2.4](https://common-changelog.org/#24-change-group), which is in the order Changed, Added, Removed, Fixed.

However, per the [Remove noise](https://common-changelog.org/#32-remove-noise) section of the docs, as @mfisher87 suggests, perhaps we omit this entry entirely. Although, I could go either way with this one because this is perhaps a borderline user-facing concern.

Reply:
Thanks; applied! I'm not as familiar with common changelog, so I need to read through it closely again (we use Keep a changelog). 

As for whether it should be there, I think it should because:
* Section 1.1 indicates maintainers/develops are consumer of the software too, and for a large open source, openly developed project, out maintainers should be expected to keep up with every PR/change.
  > Its purpose is to make it easier for consumers (and to a lesser extent contributors) to see precisely what changes have been made between two releases... The consumers of software are human beings who care about what's in the software.
  > 
  https://common-changelog.org/#11-what-is-a-changelog
* Section 3.2 indicated, since binder is a runtime environment, it's a change that should not be excluded:
  > However, changes such as the following must not be excluded:
  > * ...
  > * Changes to supported runtime environments (which may be reflected only in dotfiles)
  > 
  https://common-changelog.org/#32-remove-noise

Reply:
Cool, works for me.